x-pack/plugins/ml/server/models/data_recognizer/modules/sample_data_weblogs/ml/count_by_response_code.json

```diff
@@ -0,0 +1,37 @@
+{
+  "groups": ["kibana_sample_logs", "kibana_sample_data"],
+  "description": "Kibana sample web logs data : find anomalies in the event count by HTTP response code ",
+  "analysis_config": {
+    "bucket_span": "1h",
+    "detectors": [
+      {
+        "detector_description": "Event count by response code",
+        "function": "count",
+        "partition_field_name": "response.keyword"
+      }
+    ],
+    "influencers": ["clientip"]
+  },
+  "analysis_limits": {
+    "model_memory_limit": "10mb"
+  },
+  "data_description": {
+    "time_field": "timestamp"
+  },
+  "model_plot_config": {
+    "enabled": true
+  },
+  "custom_settings": {
+    "created_by": "ml-module-sample",
+    "custom_urls": [
+      {
+        "url_name": "Raw data",
+        "url_value": "kibana#/discover?_g=(time:(from:\u0027$earliest$\u0027,mode:absolute,to:\u0027$latest$\u0027))&_a=(index:\u002790943e30-9a47-11e8-b64d-95841ca0b247\u0027)"
```

Comment:
Is there a reason why we don't pass `response.keyword` to the Discover page? I think showing the data for the specific response code might be more useful.

Reply:
Yep, tested it and it was useful.